homeassistant/components/mystrom/__init__.py

```diff
@@ -36,7 +36,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if device_type in [101, 106, 107]:
         device = MyStromSwitch(host)
         platforms = PLATFORMS_SWITCH
-    elif device_type == 102:
+    elif device_type in [102, 105]:
```

Comment:
According to myStrom's API documentation:

> All the Bulbs request also work on the LED strip

Should be fine as "Light Effects" seems the only difference between the bulb and the strip.

Reply:
Yea I made this PR to publish the code I wanted someone else to check, as this component was updated in 2023.7.0 and there were some bugs, so I decided to dive into it. I am continuing this in the other PR.

Reply:
#96004